mobify/progressive-web-scaffold
# Enable code splitting

Split the `react`, `react-dom` and `react-router` libraries into a separate `vendor` bundle. Browsers can cache this independently from our application code, which changes more frequently.

Bundle sizes in development are:
- `main.js` => 1.5 MB
- `vendor.js` => 702 KB
- `loader.js` => 14.4 KB
- `worker.js` => 46.3 KB

Bundle sizes in production are:
- `main.js` => 549 KB
- `vendor.js` => 144 KB
- `loader.js` => 7.88 KB
- `worker.js` => 15.1 KB

 **Linked PRs**: #211 

## How to test-drive this PR
- `npm run dev`
  - Make sure www.merlinspotions.com previews as expected
- `npm run prod:build`
  - Validate that the build process completes without errors
  - Deploy the bundle to production and make sure www.merlinspotions.com loads as expected.
